fedmsg_fasclient.py

```diff
@@ -89,8 +89,32 @@ def delayed_consume():
     def action(self, messages):
         self.log.debug("Acting on %s" % pprint.pformat(messages))
 
+        # We have two playbooks: 
+        # - ``run_fasClient_simple.yml`` runs fasClient on a subset of our hosts
+        # - ``run_fasClient.yml``        runs fasClient on all our hosts
+        # So by default we run on the subset and only run on all the hosts if
+        # the message concerned a group involving sysadmins or an user that
+        # changed their ssh key.
+        playbook = 'run_fasClient_simple.yml'
+        whole_playbook = False
+        for msg in messages:
+            # If one of the groups includes a sysadmin-related group, run the
+            # complete playbook
+            if 'sysadmin-' in msg['msg'].get('group', None) \
```

Comment:
Maybe leave the ``'-'`` off the end of ``'sysadmin-'`` so we can catch the "sysadmin" group itself?